src/libraries/System.Text.Json/tests/Utf8JsonWriterTests.cs

```diff
@@ -1382,6 +1382,27 @@ public async Task WriteLargeToStream(bool formatted, bool skipValidation)
             Assert.Equal(expectedString, actualString);
         }
 
+        [Theory]
+        [InlineData(true, true)]
+        [InlineData(true, false)]
+        [InlineData(false, true)]
+        [InlineData(false, false)]
+        public async Task GrowBeyondBufferSize(bool formatted, bool skipValidation)
+        {
+            const int InitialGrowthSize = 256;
```

Comment:
This is the constant value specified here:
https://github.com/dotnet/runtime/blob/master/src/libraries/System.Text.Json/src/System/Text/Json/Writer/Utf8JsonWriter.cs#L42

I wasn't super thrilled about copying this in the test file as it will cause this test to fail if the constant is changed in the writer I think, but hopefully this is clear enough to work out why the test broke in that case.